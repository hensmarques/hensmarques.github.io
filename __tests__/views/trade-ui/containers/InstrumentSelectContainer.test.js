import React from 'react';
import InstrumentSelectContainer from '../../../../src/views/trade-ui/containers/InstrumentSelectContainer';

describe('<InstrumentSelectContainer/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<InstrumentSelectContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});