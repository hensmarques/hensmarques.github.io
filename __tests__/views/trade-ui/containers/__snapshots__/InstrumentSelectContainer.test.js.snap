// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InstrumentSelectContainer/> renders and matches the snapshot 1`] = `
<WidgetWrapper
  classNames={
    Array [
      "instrument-select",
    ]
  }
  tabs={
    Array [
      "Instruments",
    ]
  }
>
  <Connect(InstrumentSelect) />
   
</WidgetWrapper>
`;
